testing/file_test/file_test_base.cpp

```diff
@@ -386,8 +408,11 @@ static auto RunSingleTest(FileTestInfo& test, bool single_threaded,
   }
 
   // Load expected output.
+  Timer process_timer;
   test.test_result = ProcessTestFile(test_instance->test_name(),
                                      absl::GetFlag(FLAGS_autoupdate));
+  test_elapsed_ms = process_timer.elapsed_ms();
```

Comment:
I would perhaps naively have expected the timer to only capture the time taken to run the test, not the time processing inputs and autoupdating the output file. Is the goal here to determine which tests are exercising slow parts of the toolchain, or detecting slowness in the `file_test` process?

Reply:
Something slightly different from both: just which tests are slow. If we're bottlenecking on I/O, maybe we should figure that out...?

Reply:
Note, FWIW, it looks like most of the time spent is the test run itself. Removing the peripheral timings won't affect much right now; it's not where we spend time.

Reply:
OK, I think that makes sense.

I expect that we'll want to use this kind of information for other things, like determining that "making this change made the tests run X% faster", for which extra runtime spent preprocessing the test inputs ought to not be counted, because it doesn't represent time that the toolchain would take to compile code. But it sounds like that's not really an intended use case here, so I suppose that's something to be aware of when considering using this for those purposes.

Reply:
Adjusted to also track time in `::Run`, though not sure this is actually going to give you what you want.